source/solvers/fluid_dynamics_matrix_free.cc

```diff
@@ -2287,19 +2289,12 @@ FluidDynamicsMatrixFree<dim>::set_initial_condition_fd(
       this->set_nodal_values();
       this->present_solution.update_ghost_values();
 
-      // Create a pointer to the current viscosity model
-      std::shared_ptr<RheologicalModel> viscosity_model =
-        this->simulation_parameters.physical_properties_manager.get_rheology();
-
-      // Keep in memory the initial viscosity
-      const double viscosity_end = viscosity_model->get_kinematic_viscosity();
-
-      // Set it to the initial condition viscosity
-      viscosity_model->set_kinematic_viscosity(
-        this->simulation_parameters.initial_condition->kinematic_viscosity);
+      const double viscosity_end =
```

Comment:
What is viscosity end ? Could you explain what it is or maybe give a more precise variable name ? End of what?

Reply:
This part of the code has now being updated. Before, we needed to set the kinematic viscosity of the operators when using viscous or ramp initial conditions, now they look very similar to the matrix-based code and use the rheological model of the physical properties.